contrib/mesos/pkg/scheduler/plugin_test.go

```diff
@@ -362,28 +371,30 @@ func newTaskStatusForTask(task *mesos.TaskInfo, state mesos.TaskState) *mesos.Ta
 	}
 }
 
-// Test to create the scheduler plugin with an empty plugin config
-func TestPlugin_New(t *testing.T) {
-	assert := assert.New(t)
+type LaunchedTask struct {
```

Comment:
refactoring things into a separate `schedulertest` package sounds good (let's avoid adding another `test` package to avoid confusion)